plugins/modules/keycloak_client.py

```diff
@@ -581,6 +582,14 @@
                     - For OpenID-Connect clients, client certificate for validating JWT issued by
                       client and signed by its key, base64-encoded.
 
+            x509.subjectdn:
+                description:
+                    - For OpenID-Connect clients, subject which will be used to authenticate the client.
```

Comment:
This is a new suboption, so it needs to be announced as added in the next release. Also, you need to specify a type. Is this a string? Then use `type: str`.
```suggestion
                    - For OpenID-Connect clients, subject which will be used to authenticate the client.
                type: ...
                version_added: 9.5.0
```

Reply:
None of the other suboptions under `attributes` have a type specified, I assumed these was merely examples, is this not correct?

Reply:
Nope, these are not examples, they are the source for the module documentation, which will be rendered to HTML and published later to the official documentation site, in the case of this module, here:

https://docs.ansible.com/ansible/latest/collections/community/general/keycloak_client_module.html#ansible-collections-community-general-keycloak-client-module

So, whilst other suboptions might not specify types, in general the more information we can include here, the more complete our user documentation is going to be.